frontend/src/components/DataTable.vue

```diff
@@ -217,7 +254,12 @@ export default defineComponent({
 
       rows = rows.filter(row => {
         if (search.value !== '') {
-          const result = config.search.function(row, search.value)
+          if (typeof config.search.handler !== 'function') {
```

Comment:
Could we use prop validation for this?

Reply:
Mmmm, not sure how, its a nested field under `config`

Reply:
https://stackoverflow.com/questions/43481866/vuejs-object-props-validation#43482133

Reply:
haha, ok lol!